lib/ts/types.ts

```diff
@@ -358,3 +367,5 @@ export type Navigate =
           goBack: () => void;
       }
     | NavigateFunction;
+
+export type UserContext = Record<string, unknown>;
```

Comment:
I think it's better to put `Record<string, any>` cause with `unknown`, devs have to first explicitly check the type of it using `typeof` or something and then use the value. So it can get annoying for them. What do you think?